recipes/ccache-toolchain/activate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export _OLD_PATH=${PATH}
```

Comment:
Yeah, me too. But I was bitten by that before. Conda just list the files and the order is undefined (at least on ext4 filesystems)